bin/cli.js

```diff
@@ -3,18 +3,25 @@
 
 const fs = require('fs');
 const path = require('path');
-
```

Comment:
There is no need to remove this line, it was separating node.js module requires from npm module requires.